libs/database/prisma/schema.prisma

```diff
@@ -6,7 +6,7 @@ generator client {
 }
 
 datasource db {
-  provider = "sqlite"
+  provider = "postgresql"
   url      = env("DATABASE_URL")
```

Comment:
Should it be mentioned to set
DATABASE_URL=postgres://postgres:password@localhost:5432
in the .env file?
in the libs/database/readme?

Reply:
Good call; added a `.env.example` file and mentioned it in the readme 🤠